examples/awesome-vault-service/config/config.toml

```
@@ -42,7 +42,7 @@ delegation_manager_address = "0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0"
 rewards_coordinator_address = "0xa51c1fc2f0d1a1b8494ed1fe312d7c3a78ed91c0"
 permission_controller_address = "0x59b670e9fa9d0a427751af201d676719a970857b"
 
-ecdsa_private_key_store_path = "keys/test.ecdsa.key.json"
+ecdsa_signer = {keystore_path = "keys/test.ecdsa.key.json", keystore_password = ""}
```

Let's move to the multi-line syntax:

```suggestion
[ecdsa_signer]
keystore_path = "keys/test.ecdsa.key.json"
keystore_password = ""
```

---

Done in [20f1148](https://github.com/Layr-Labs/eigensdk-go/pull/727/commits/20f1148486d015bd69679b4d44edfa3789b738e4)